### PR TITLE
[ppdiffusers] add AnimateAnyone inference example

### PR DESCRIPTION
**Pull request by Tsaiyue**

### CODE REVIEW

File: ppdiffusers/examples/AnimateAnyone/src/models/mutual_self_attention.py [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+        fusion_blocks = fusion_blocks
+        num_images_per_prompt = num_images_per_prompt
+        dtype = dtype
+        if do_classifier_free_guidance:
+            uc_mask = (
+                paddle.to_tensor(
+                    data=[1] * batch_size * num_images_per_prompt * 16 + [0] * batch_size * num_images_per_prompt * 16,
+                    dtype="float32",
+                )
+                .to(device)
```
Comment (westfish) [inline ppdiffusers/examples/AnimateAnyone/src/models/mutual_self_attention.py]: device相关的可以去除

File: ppdiffusers/examples/AnimateAnyone/src/models/mutual_self_attention.py [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+                .astype(dtype="bool")
+            )
+        else:
+            uc_mask = (
+                paddle.to_tensor(data=[0] * batch_size * num_images_per_prompt * 2, dtype="float32")
+                .to(device)
```
Comment (westfish) [inline ppdiffusers/examples/AnimateAnyone/src/models/mutual_self_attention.py]: 去除

File: ppdiffusers/examples/AnimateAnyone/src/models/mutual_self_attention.py [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+                        hidden_states_c = hidden_states_uc.clone()
+                        _uc_mask = uc_mask.clone()
+                        if hidden_states.shape[0] != _uc_mask.shape[0]:
+
+                            _uc_mask = (
+                                paddle.to_tensor(
+                                    data=[1] * (hidden_states.shape[0] // 2) + [0] * (hidden_states.shape[0] // 2),
+                                    dtype="float32",
+                                )
+                                .to(device)
```
Comment (westfish) [inline ppdiffusers/examples/AnimateAnyone/src/models/mutual_self_attention.py]: 去除

File: ppdiffusers/examples/AnimateAnyone/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+通过 `git clone` 命令拉取 PaddleMIX 源码，并安装必要的依赖库。请确保你的 PaddlePaddle 框架版本在 2.6.0 之后，PaddlePaddle 框架安装可参考 [飞桨官网-安装](https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/install/pip/linux-pip.html)。
+
+```bash
+# 克隆 PaddleMIX 仓库
+git clone https://github.com/PaddlePaddle/PaddleMIX
+
+# 安装2.6.0版本的paddlepaddle-gpu，当前我们选择了cuda12.0的版本，可以查看 https://www.paddlepaddle.org.cn/ 寻找自己适合的版本
+python -m pip install paddlepaddle-gpu==2.6.0.post120 -f https://www.paddlepaddle.org.cn/whl/linux/mkl/avx/stable.html
+
+# 进入consistency_distillation目录
```
Comment (westfish) [inline ppdiffusers/examples/AnimateAnyone/README.md]: 改一下

File: ppdiffusers/examples/AnimateAnyone/README.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+# 安装新版本ppdiffusers
+pip install https://paddlenlp.bj.bcebos.com/models/community/junnyu/wheels/ppdiffusers-0.24.0-py3-none-any.whl --user
+
+# 安装其他所需的依赖, 如果提示权限不够，请在最后增加 --user 选项
+pip install -r requirements.txt
+```
+
+## 3. 模型下载
+
+运行以下自动下载脚本，通过 [Huggingface](https://huggingface.co/InstantX/InstantID) 下载 AnimateAnyone 相关模型权重文件，模型权重文件将存储在`./pretrained_weights`下面。
```
Comment (westfish) [inline ppdiffusers/examples/AnimateAnyone/README.md]: 链接错了，改一下

Comment (Tsaiyue) [reply ppdiffusers/examples/AnimateAnyone/README.md]: All the above have done.
Thks 4 ur review.

**Review by westfish — APPROVED** [review]
LGTM

**Pull request merged**